package/lib/src/beam_interceptor.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:beamer/beamer.dart';
+import 'package:flutter/widgets.dart';
+
+/// IDK LOL
+class BeamInterceptor {
+  /// Creates a [BeamInterceptor] with defined properties.
+  ///
+  /// [pathPatterns] and [intercept] must not be null.
```

Comment:
This doc comment is referring to wrong variable names

Reply:
✅